packages/atomic/src/components/result-template-components/atomic-result-text/atomic-result-text.tsx

```diff
@@ -60,23 +78,23 @@ export class AtomicResultText {
         ></atomic-component-error>
       );
     }
-    const value = ResultTemplatesHelpers.getResultProperty(
-      this.result,
-      this.field
-    );
 
-    if (value === null) {
+    if (!this.resultValue && !this.default) {
```

Comment:
Since `resultValue` is a getter with a little bit of logic, I would have a small preference for executing it only once. E.g.:
```ts
const resultValue = this.resultValue;
if (!resultValue && !this.default) {
  ...
```